app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,102 @@
+
+    <string name="dialog_developer_settings_title">Developer Mode deaktiviert</string>
+    <string name="dialog_developer_settings_message">Aktivieren Sie den Developer Mode, indem Sie 7 (sieben) Mal auf die Build-Nummer Ihres Telefons klicken.</string>
+
+    <string name="dialog_developer_settings_nougat_title">Developer Settings</string>
+    <string name="dialog_developer_settings_nougat_message">Bitte stellen Sie sicher, dass die Entwicklereinstellungen aktiviert sind, bevor Sie versuchen, den Standort festzulegen, um Probleme unter Nougat zu vermeiden.
+        Sie können dies tun, indem Sie 7 (sieben) Mal auf die Build-Nummer Ihres Telefons klicken.</string>
+
+    <!--about-->
+    <string name="about_overview_logo_desc">About overview app logo</string>
+
+    <string name="about_contribute_card_title">Mitwirken</string>
+    <string name="about_contribute_title">Fehler melden</string>
+    <string name="about_contribute_desc">Fehler melden oder neue Features anfragen</string>
+
+    <string name="about_developer_image_desc">Bild des Autors</string>
+    <string name="about_developer_title">wesaphzt</string>
+    <string name="about_developer_desc">Developer</string>
+    <string name="about_developer_card_title">Developer</string>
+
+    <string name="about_version_title">Version</string>
+    <string name="about_version_image_desc">Version image</string>
+
+    <string name="about_license_title">Lizenz</string>
+    <string name="about_license_desc">GPL v3.0 Lizenz</string>
+    <string name="about_license_image_desc">License image</string>
+
+    <string name="about_source_title">Quellcode</string>
+    <string name="about_source_image_desc">Quellcode image</string>
+
+    <string name="about_bug_report_image_desc">Fehler melden image</string>
+
+    <!--message-->
+    <string name="settings_randomize_toast">Wenn der Dienst gerade läuft, starten Sie ihn neu, um die Einstellung zu übernehmen.
+        Dies kann die Bildschirmtemperatur und die Lokalisierung beeinträchtigen.</string>
+
+    <!--widget-->
+    <string name="widget_start_text">Start</string>
+    <string name="widget_stop_text">Stop</string>
+
+    <string name="widget_prefs_service_id">SERVICE_STATUS</string>
+    <string name="widget_location_image_desc">Widget location image</string>
+
+    <!--intro-->
+    <string name="slider_page_one_title">@string/app_name</string>
+    <string name="slider_page_one_desc">Der perfekte Begleiter, um die Privatsphäre und Sicherheit Ihres Telefonstandorts zu schützen, indem er Ihren Standort an jeden Ort der Welt verschleiert.</string>
+    <string name="slider_page_two_title">Beschränken Sie Ihre Daten</string>
+    <string name="slider_page_two_desc">Viele Apps können im Hintergrund wiederholte und unnötige Standortabfragen machen und diese Daten oft an die Cloud zurücksenden.
+        Standortdaten sind persönlich und potenziell sehr sensibel. Diese App kann helfen, Ihre Privatsphäre und Daten zu schützen.</string>
```

Comment:
"potenziell sehr sensibel" sounds a bit weird in German. I would suggest "potenziell sensibel"